src/app/sc2/models/match.py

```diff
@@ -73,3 +75,9 @@ def lookup_open(cls):
     @classmethod
     def _get_kind(cls):
         return "SCII_MatchModel"
+
+    def _post_put_hook(self, future):
+        """ post put hook """
+        # future.check_success()
+        # from app.sc2.domain.slack import update_channel_topic_with_open_games
+        # TODO:slack add channel topic setting here
```

Comment:
Fyi I'd have to check but this might get called whenever a replay is uploaded. Might want to just call this if match newly created.